Gemfile.lock

```diff
@@ -254,7 +254,7 @@ GEM
     mocha (1.1.0)
       metaclass (~> 0.0.1)
     modernizr-rails (2.7.1)
-    mongo (2.2.7)
+    mongo (2.3.0)
```

Comment:
This is causing the CI failures. We've had this problem before; we should decide whether we want to fix it, or pin `mongo` to `< 2.3.0`.